src/coreComponents/mesh/generators/WellGeneratorBase.cpp

```diff
@@ -141,9 +144,10 @@ void WellGeneratorBase::generateWellGeometry( )
   // make sure that the perforation locations are valid
   checkPerforationLocationsValidity();
 
-  if( getLogLevel() >= 1 )
+  if( getLogLevel() >= 1 && logger::internal::rank == 0 )
```

Comment:
The `internal` may clue you into guessing it's meant for internal purposes 🤣 😉 🤗 
Please use `MpiWrapper::comRank()` instead.

Reply:
Hi @TotoGaz, about these `logger::internal`, I'm going to refactor them all by hiding them in a `Logger` class and provide an assessor (we discussed of that quite a long time ago). There are like 30 of them currently ;)

Reply:
No pb 😉